index.html

```diff
@@ -43,5 +45,195 @@ <h1>Hey There. I'm Carlyle</h1>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+         </div> 
+        <div class="card card5">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card6">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="info">
+    <div class="about">
+      <h2 class="am-title">About Me</h2>
+      <p class="am-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] [Required] Again, let's try to maintain the Figma design. The margins in your project differ from that of Figma's. What do you think?
![image](https://user-images.githubusercontent.com/45668340/148661900-f530a2e0-f306-4f38-9f3d-40b336c62531.png)

Compare with that of Figma's below:point_down:
![image](https://user-images.githubusercontent.com/45668340/148661916-11ccf2b9-8c0c-49f5-ad3a-71b896b2146c.png)
- [ ] You seem to have forgotten the background image in the design.
![image](https://user-images.githubusercontent.com/45668340/148661952-516651de-5840-4817-aa20-4f7ae93bf823.png)
## Figma's 
![image](https://user-images.githubusercontent.com/45668340/148661959-5c2ae326-347d-4846-b2de-78a88aebd7f6.png)